astropy/coordinates/representation.py

```diff
@@ -354,23 +345,7 @@ def lat(self):
         """
         return self._lat
 
-    @property
-    def distance(self):
-        """
-        The distance from the origin to the point(s).
-        """
-        return self._distance
-
-    def represent_as(self, other_class):
-        # Take a short cut if the other class is a spherical representation
-        if other_class is PhysicsSphericalRepresentation:
-            return PhysicsSphericalRepresentation(phi=self.lon,
-                                                  theta=90 * u.deg - self.lat,
-                                                  r=self.distance)
-        elif other_class is UnitSphericalRepresentation:
-            return UnitSphericalRepresentation(lon=self.lon, lat=self.lat)
-        else:
-            return super(SphericalRepresentation, self).represent_as(other_class)
+    # TODO: implement represent_as for efficient transformations
```

Comment:
`represent_as` is in fact defined a bit further down, so remove this comment? (Or clarify it and move it further down).